src/app/community/_components/AuthorCard.tsx

```diff
@@ -12,39 +13,71 @@ interface AuthorCardProps {
   nickname: string;
   dateText: string;
   userImage: string | null;
+  userId: number;
   onClickPopOver: () => void;
   onClosePopOver: () => void;
   isOpenPopOver: boolean;
   popOverOptions: {
     label: string;
-    onClick: (e: React.MouseEvent<HTMLDivElement>) => void;
+    onClick?: () => void;
   }[];
 }
 
 export default function AuthorCard({
   nickname,
   dateText,
   userImage,
+  userId,
   onClickPopOver,
   onClosePopOver,
   isOpenPopOver,
   popOverOptions,
 }: AuthorCardProps) {
+  const userProfileCardRef = useRef<HTMLDivElement>(null);
+  const [isOpenUserCard, setIsOpenUserCard] = useState(false);
+  const [isHovering, setIsHovering] = useState(false);
+
+  const handleOpenProfile = () => {
+    setIsOpenUserCard(true);
+  };
+
+  const handleCloseProfile = () => {
+    setIsOpenUserCard(false);
+  };
+
   return (
-    <div className={cn('container')}>
-      <ProfileImage profileImage={userImage} />
+    <div
+      className={cn('container')}
+      onClick={(e) => e.stopPropagation()}
+      onMouseEnter={() => setIsHovering(true)}
+      onMouseLeave={() => setIsHovering(false)}
+    >
+      <div
+        onMouseEnter={handleOpenProfile}
+        onMouseLeave={handleCloseProfile}
+        className={cn('user-profile')}
+        ref={userProfileCardRef}
+      >
+        <ProfileImage profileImage={userImage} />
+        <UserProfileCard isOpenProfileCard={isOpenUserCard} userId={userId} />
+      </div>
       <div className={cn('info-textbox')}>
-        <p className={cn('user-name')}>{nickname}</p>
+        <p className={cn('user-name')} onMouseEnter={handleOpenProfile} onMouseLeave={handleCloseProfile}>
```

Comment:
```suggestion
        <p className={cn('user-name')} {...profileHandlers}>
```
만약 추가 작업을 한다면 여기도 이런식으로 사용 가능하겠네요